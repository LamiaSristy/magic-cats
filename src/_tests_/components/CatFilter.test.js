/*eslint-disable*/
import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom/extend-expect';
import CatSearch from '../../components/CatFilter';

const setup = () => shallow(<CatSearch />);

describe('CatSearch component', () => {
  let component;
  beforeEach(() => {
    component = setup();
  });

  it('Should render', () => {
    const element = component.find('.search-input');
    expect(element).toBeVisible;
  });

  it('Should render', () => {
    const element = component.find('.search-input');
    expect(element.length).toEqual(1);
  });

  it('Should render', () => {
    const element = component.find('.search-input');
    expect(element).toBeTruthy;
  });

  it('Should render', () => {
    const element = component.find('.search');
    expect(element).toBeFalsy;
  });

  test('renders the search-input in the page', () => {
    const item = component.find('.search-input');
    expect(item).toMatchSnapshot();
  });
});
