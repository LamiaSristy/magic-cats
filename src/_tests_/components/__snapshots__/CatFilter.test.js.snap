// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CatSearch component renders the loading-wrap in the page 1`] = `ShallowWrapper {}`;

exports[`CatSearch component renders the search-input in the page 1`] = `ShallowWrapper {}`;
